2025-08-23 01:38:41,476 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://scaling-xylophone-7v47xjqj967vc79g-8501.app.github.dev/
2025-08-23 01:50:20,080 - api_endpoints.pdf_api - INFO - upload_pdf - 24- File saved successfully: upload/Identification documents(1).pdf
2025-08-23 01:50:21,474 - api_endpoints.pdf_api - ERROR - upload_pdf - 30- Error while processing PDF
Traceback (most recent call last):
  File "/workspaces/Chatbot/chatbot/api_endpoints/pdf_api.py", line 27, in upload_pdf
    add_pdf_data(pdf_file_path=file_path)
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 60, in add_pdf_data
    if process_pdf_data(pdf_data_df):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 43, in process_pdf_data
    pdf_data_collection.add(
TypeError: Collection.add() got an unexpected keyword argument 'metadata'
2025-08-23 02:07:57,972 - api_endpoints.pdf_api - INFO - upload_pdf - 24- File saved successfully: upload/ppa_Letter (1).pdf
2025-08-23 02:07:58,912 - api_endpoints.pdf_api - ERROR - upload_pdf - 30- Error while processing PDF
Traceback (most recent call last):
  File "/workspaces/Chatbot/chatbot/api_endpoints/pdf_api.py", line 27, in upload_pdf
    add_pdf_data(pdf_file_path=file_path)
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 57, in add_pdf_data
    if process_pdf_data(pdf_data_df):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 40, in process_pdf_data
    pdf_data_collection.add(
TypeError: Collection.add() got an unexpected keyword argument 'metadata'
2025-08-23 02:24:52,241 - api_endpoints.pdf_api - INFO - upload_pdf - 24- File saved successfully: upload/ppa_Letter (1).pdf
2025-08-23 02:24:52,676 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 03:28:39,724 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 03:28:40,891 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:05:18,409 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:05:22,494 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:05:39,094 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:06:13,883 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:06:54,325 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:07:50,335 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:07:51,178 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event ClientCreateCollectionEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:07:51,283 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionQueryEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:07:51,284 - api_endpoints.web_api - INFO - query_messages - 52- Web query 'https://www.timeanddate.com/' returned 10 results
2025-08-23 04:08:21,103 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://www.timeanddate.com/
2025-08-23 04:08:21,779 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event ClientCreateCollectionEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:08:21,809 - api_endpoints.web_api - INFO - query_messages - 52- Web query 'https://www.timeanddate.com/' returned 10 results
2025-08-23 04:38:38,947 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://google.com
2025-08-23 04:38:39,577 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:39:20,492 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://google.com
2025-08-23 04:46:22,816 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://google.com
2025-08-23 04:46:23,453 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:51:03,991 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 04:51:19,394 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 04:51:31,086 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 04:51:31,502 - api_endpoints.pdf_api - INFO - query_messages - 36- Received query: summarize it
2025-08-23 04:51:31,503 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event ClientCreateCollectionEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:51:31,516 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionQueryEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 04:51:31,519 - api_endpoints.pdf_api - INFO - query_messages - 48- PDF query 'summarize it' returned 0 result
2025-08-23 04:51:47,896 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 04:52:14,763 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 04:52:34,483 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 05:34:33,180 - api_endpoints.web_api - INFO - scrape_webdata - 17- Scraping task queued for URL: https://semprebarca.com/barcelona-rejected-e80m-bids-from-chelsea-and-manchester-united-for-22-year-old-player/
2025-08-23 05:34:38,012 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 08:38:07,563 - api_endpoints.pdf_api - INFO - upload_pdf - 24- File saved successfully: uploads/sω² @ a Glance v2.0 (1).pdf
2025-08-23 08:38:12,272 - chromadb.telemetry.product.posthog - ERROR - _direct_capture - 59- Failed to send telemetry event CollectionAddEvent: capture() takes 1 positional argument but 3 were given
2025-08-23 08:38:12,272 - api_endpoints.pdf_api - ERROR - upload_pdf - 30- Error while processing PDF
Traceback (most recent call last):
  File "/workspaces/Chatbot/chatbot/api_endpoints/pdf_api.py", line 27, in upload_pdf
    add_pdf_data(pdf_file_path=file_path)
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 57, in add_pdf_data
    if process_pdf_data(pdf_data_df):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/workspaces/Chatbot/chatbot/pdf_data_collector.py", line 48, in process_pdf_data
    print(f"Stored {num_of_docs} chunk{'' if num_of_chunks < 2 else 's'} in ChromaDB.")
                    ^^^^^^^^^^^
NameError: name 'num_of_docs' is not defined
